cypress/e2e/pages/profile.cy.ts

```diff
@@ -1,7 +1,45 @@
 /// <reference types="cypress" />
 describe('Profile page', () => {
   it('Should visit the profile page and check the title', () => {
```

Comment:
A better name would be "Should visit the profile page and check the title"

Reply:
done